spec/support/factory_bot.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
```

Comment:
you should uncomment this line `Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }` in the `rails_helper.rb` to be able to load support files